frontend/interfaces/query.ts

```diff
@@ -1,7 +1,6 @@
 import { IFormField } from "./form_field";
 import { IPack } from "./pack";
-import { ISchedulableQuery } from "./schedulable_query";
```

Comment:
todo: can we remove /interfaces/query.ts and replace with /interfaces/schedulable_query.ts